kt/api-generator/src/main/resources/api.json

```diff
@@ -2,10 +2,10 @@
 	"header": {
 		"version_major": 4,
 		"version_minor": 2,
-		"version_patch": 1,
+		"version_patch": 2,
 		"version_status": "stable",
-		"version_build": "custom_build",
-		"version_full_name": "Godot Engine v4.2.1.stable.custom_build"
+		"version_build": "official",
```

Comment:
Why is it suddenly `official`?

Reply:
I think it's because I downloaded the api.json they provide instead of generating it from our own binary.

Reply:
I would probably generate it locally